src/app/public/modules/repeater/fixtures/repeater.component.fixture.html

```diff
@@ -1,17 +1,35 @@
 <sky-repeater
   [activeIndex]="activeIndex"
+  [enableActive]="enableActive"
```

Comment:
I'm not crazy about `enableActive` as the name for this property. Any suggestions?

Reply:
Moot point now. property removed.